app/controllers/items_controller.rb

```diff
@@ -0,0 +1,47 @@
+# app/controllers/items_controller.rb
+class ItemsController < ApplicationController
+  before_action :set_todo
+  before_action :set_todo_item, only: [:show, :update, :destroy]
+
+  # GET /todos/:todo_id/items
+  def index
+    json_response(@todo.items)
+  end
+
+  # GET /todos/:todo_id/items/:id
+  def show
+    json_response(@item)
+  end
+
+  # POST /todos/:todo_id/items
+  def create
+    @todo.items.create!(item_params)
+    json_response(@todo, :created)
+  end
+
+  # PUT /todos/:todo_id/items/:id
+  def update
+    @item.update(item_params)
+    head :no_content
```

Comment:
what would be the functionality of `head :no_content`?

Reply:
this returns an empty header (HTTP status code 204) in response to the request made to the item